src/tokens/widths.json

```diff
@@ -0,0 +1,36 @@
+{
+  "width": {
+    "base": {
+      "value": "{spacing.xxl.value}",
+      "comment": "64px"
+    },
+    "xxs": {
+      "value": "{width.base.value}",
+      "comment": "64px"
+    },
+    "xs": {
+      "value": "({width.base.value} * 2)",
+      "comment": "128px"
+    },
+    "sm": {
+      "value": "({width.base.value} * 3)",
+      "comment": "192px"
+    },
+    "md": {
+      "value": "({width.base.value} * 4)",
+      "comment": "256px"
+    },
+    "lg": {
+      "value": "({width.base.value} * 5)",
+      "comment": "320px"
+    },
+    "xl": {
+      "value": "({width.base.value} * 6)",
+      "comment": "384px"
+    },
+    "xxl": {
+      "value": "({width.base.value} * 8)",
```

Comment:
Same here, I would think about either moving the multiplication inside the curly brackets/expression (if possible), or doing the math and hard-coding the value here. The goal being that any of the compiled tokens would have the final values as strings instead of math expressions which is how they are currently being output.